tech_reports/LLMs/llms.md

```diff
@@ -81,10 +81,33 @@ Authors:
     - 1d, 2d, dram-sharded, ...
   - Implicitly padding weights in program config for matmuls
 ### 4.5 Accuracy
-  - How we measure it (PCC, perplexity, top-1/top-5, end-user tests, benchmarking)
-  - How much PCC is enough? Rules of thumb.
-  - Accuracy tests
-  - Debugging PCC issues
+
+While we work on maximizing the performance of large language models on Tenstorrent hardware, we must also ensure that the models are functionally correct and that they produce outputs of the expected quality. The subsections below will describe our methods for evaluating the accuracy (also referred to as functionality or correctness for our purposes) of a given model and how to debug issues pertaining to this.
+
+#### Accuracy Testing
+
```

Comment:
I think we should add a list of various accuracy metrics and one sentence summary of what they are and mention that we use all these technique in different tests, e.g. pcc, perplexiy, top1/top5, occular testing, benchmark eval scores).

Reply:
Added the list of accuracy metrics